python/dgl/data/utils.py

```diff
@@ -216,8 +216,11 @@ def extract_archive(file, target_dir, overwrite=False):
     elif file.endswith('.gz'):
         import gzip
         import shutil
+        fname = os.path.basename(file)
```

Comment:
Does the change in this function change the behavior for other datasets?

Reply:
Maybe I should do it in this specific dataset.